CONTRIBUTING.md

```diff
@@ -1,86 +1,87 @@
-It means that if your change break the CI in your PR, it will not be merged until the breaking change are addressed.
-It also means that adding a new feature or behavior means adding a associated test.
-Make sure to check the results for yourself, and ask for help if needed.
+F3D has pretty extensive continuous integration trying to cover all usecases for F3D.
+It means that if the change in your pull request breaks continuous integration, it will not be merged until it passes successfully.
+It also means that adding a new feature or behavior means adding an associated test.
+Make sure to check the results for yourself and ask for help if needed.
 
 F3D continuous integration will also check the coverage as it is a good way to evaluate if new features are being tested or not.
 When adding code to F3D, always try to cover it by adding/modifying [tests](doc/dev/TESTING.md).
 
-F3D continuous integration also check formatting using clang-format and will inform you if changes needs to be made.
-However, some [formatting rules](doc/dev/CODING_STYLE.md) are not enforced by clang-format and will be checked during the review process.
+F3D continuous integration also checks formatting using clang-format and will inform you if changes needs to be made.
+Some [formatting rules](doc/dev/CODING_STYLE.md) are not enforced by clang-format and will be checked during the review process.
 
-When making changes to the libf3d public API, the CI will warn about making related changes to the bindings. This is required in order to merge the PR.
+When making changes to the libf3d public API, continuous integration will warn about making related changes to the bindings.
+This is required in order to merge the pull request.
 
 ## Changelog
 
 Whenever you make significant changes, either bugfixes or features, please update the [changelog](doc/CHANGELOG.md) "Ongoing development" section.
-There are three subsections. "For F3D Users" are changes that impact actual F3D users and related to F3D behaviors changing. "For developers" are change
-that impact developers that build F3D. "For F3D packagers" are for changes that impact anyone packaging F3D, eg: new CMake options.
+There are three subsections. "For F3D Users" are changes that impact actual F3D users and are related to F3D behaviors changing. "For developers" are changes
+that impact developers that build F3D. "For F3D packagers" are changes that impact anyone packaging F3D, eg: new CMake options.
 
-## F3D architecture
+## F3D Architecture
 
-F3D is separated in different components:
-- The F3D application, in the application folder.
-- The libf3d, in the library folder.
-- The VTKExtensions in the library/VTKExtensions folder.
-- The bindings, python, java and webassembly, in the respective directories.
+F3D is divided into different components:
+- The f3d executable, in the application folder.
+- The libf3d library, in the library folder.
+- The VTKExtensions, in the vtkext folder.
```

Comment:
Extensions to vtk, in the vtkext folder.

Reply:
Updated in commit c1633c7.